java/src/jmri/managers/ProxyTurnoutManager.java

```diff
@@ -205,7 +205,7 @@ public void setDefaultClosedSpeed(@Nonnull String speed) throws jmri.JmriExcepti
             try {
                 ((TurnoutManager) m).setDefaultClosedSpeed(speed);
             } catch (jmri.JmriException ex) {
-                log.error(ex.toString());
+                log.error("Manager {} {}", m, ex.getMessage() );
```

Comment:
I don't think the managers have a `toString()` method that gives anything useful except the class name. And the class name is shown anyway in the log. So I suggest removing the manager from the message.